src/main/java/org/alliancegenome/curation_api/model/entities/LoggedInPerson.java

```diff
@@ -38,4 +44,9 @@ public class LoggedInPerson extends Person {
 
     @JsonView({View.FieldsOnly.class})
     private String apiToken;
+    
+    @Type(type = JsonTypes.JSON_BIN)
+    @Column(columnDefinition = JsonTypes.JSON_BIN)
+    private JsonNode userSettings;
```

Comment:
@markquintontulloch I think we need this type to be a `Map<String, Object>` instead of `JsonNode`

Reply:
Maybe it doesn't matter... but I would like to see how they both work.

Reply:
I've changed it to Map<String,Object>